project_mis_report/models/project_project.py

```diff
@@ -0,0 +1,67 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, _
+from odoo.exceptions import ValidationError
+from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    mis_report_template_id = fields.Many2one('mis.report', string="Mis Report Template")
+    mis_report_instance_id = fields.Many2one('mis.report.instance', string="Mis Report")
+    is_group_account_manager = fields.Boolean(compute='_compute_is_group_account_manager')
+    is_group_project_manager = fields.Boolean(compute='_compute_is_group_project_manager')
+
+    def _compute_is_group_project_manager(self):
+        for record in self:
+            record.is_group_project_manager = True if record.user_has_groups(
+                'project.group_project_manager') else False

```

Comment:
This is inefficient, because user_has_groups is actually independent of the record:
```suggestion
        self.write({
            "is_group_project_manager": self.user_has_groups("project.group_project_manager"),
        })
```